Example/JustLog/AppDelegate.swift

```diff
@@ -91,40 +91,39 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
+        let regexRuleList = "[{\"pattern\": \"(name) = \\\\\\\\*\\\"(.*?[^\\\\\\\\]+)\", \"minimumLogLevel\": \"warning\"}, {\"pattern\": \"(token) = \\\\\\\\*\\\"(.*?[^\\\\\\\\]+)\", \"minimumLogLevel\": \"warning\"}]".data(using: .utf8)
+        let sanitizationExeceptionList = "[{\"value\": \"Dan Jones\"}, {\"value\": \"Jack Jones\"}]".data(using: .utf8)
         
         logger.sanitize = { message, type in
             var sanitizedMessage = message
             
             guard let ruleList = try? decoder.decode([RegexListReponse].self, from: regexRuleList!) else { return "sanitizedMessage" }
             guard let exceptionList = try? decoder.decode([ExceptionListResponse].self, from: sanitizationExeceptionList!) else { return "sanitizedMessage" }
             
-                for value in ruleList {
-                    if (value.minimumLogLevel >= type.rawValue) {
-                        if let regex = try? NSRegularExpression(pattern: value.pattern , options: NSRegularExpression.Options.caseInsensitive) {
-                            
-                            let range = NSRange(sanitizedMessage.startIndex..<sanitizedMessage.endIndex, in: sanitizedMessage)
-                            let matches = regex.matches(in: sanitizedMessage, options: [], range: range)
-                            
-                            sanitizedMessage = self.redactValues(message: sanitizedMessage, loggerExeptionList: exceptionList, matches: matches)
-                        }
+            for value in ruleList {
+                if (value.minimumLogLevel >= type.rawValue) {
+                    if let regex = try? NSRegularExpression(pattern: value.pattern , options: NSRegularExpression.Options.caseInsensitive) {
+                        
+                        let range = NSRange(sanitizedMessage.startIndex..<sanitizedMessage.endIndex, in: sanitizedMessage)
+                        let matches = regex.matches(in: sanitizedMessage, options: [], range: range)
+                        
+                        sanitizedMessage = self.redactValues(message: sanitizedMessage, loggerExeptionList: exceptionList, matches: matches)
                     }
+                }
             }
             return sanitizedMessage
         }
         
         // default info
-        logger.defaultUserInfo = ["application": "JustLog iOS Demo",
-                                  "environment": "development",
-                                  "session": sessionID]
-        
+        Log.defaultUserInfo = ["application": "JustLog iOS Demo",
+                               "environment": "development",
+                               "session": sessionID]
         
         // custom keys
-        logger.logTypeKey = "logtype"
-        logger.appVersionKey = "app_version"
-        logger.iosVersionKey = "ios_version"
-        logger.deviceTypeKey = "ios_device"
+        LogKeys.logTypeKey = "logtype"
```

Comment:
I see `LogKeys` is a struct with static vars.
For the usual reasons about static vars, is best to repurpose this to use instance vars.